portal-api/src/v1/validation/fields/cover-end-date.js

```diff
@@ -18,16 +15,23 @@ module.exports = (submittedValues, deal, errorList) => {
   } = submittedValues;
 
   if (isReadyForValidation(deal, submittedValues)) {
-    const {
-      coverDayValidation,
-      coverMonthValidation,
-      coverYearValidation
-    } = coverDatesValidation(coverEndDateDay, coverEndDateMonth, coverEndDateYear);
+    const { coverDayValidation, coverMonthValidation, coverYearValidation } = coverDatesValidation(
+      coverEndDateDay,
+      coverEndDateMonth,
+      coverEndDateYear,
+    );
 
-    if (dateHasAllValues(coverEndDateDay, coverEndDateMonth, coverEndDateYear)) {
-      const formattedDate = `${coverEndDateYear}-${coverEndDateMonth}-${coverEndDateDay}`;
-      const nowDate = moment().format('YYYY-MM-DD');
-      if (moment(formattedDate).isBefore(nowDate)) {
+    const coverEndDateHasAllValues =
+      hasValue(coverEndDateDay) && hasValue(coverEndDateMonth) && hasValue(coverEndDateYear);
```

Comment:
As per `supply-contract-conversion-rate-date-is-required-for-non-GBP.js` (what a mouthful!), if we're only caring about dates, lets work as `startOfDay` throughout please for simplicity -- what do you think?

Also same comments on this file as `supply-contract-conversion-rate-date-is-required-for-non-GBP.js`